content/materials/overviews/10v10.vmat

```diff
@@ -2,5 +2,5 @@
 {
 	"Shader"	"ui.vfx"
 	"F_TRANSLUCENT"	"0"
-	"Texture"	"materials/overviews/10v10.tga"
+	"Texture"	"materials/overviews/oaa.tga"
```

Comment:
10v10 has a different minimap since the map is shaped differently

Reply:
This pull makes 10v10.vmap the same as others.
Looked in to "Add borders around the map to prevent fountain camping #2399 " and seemed they (vmaps) were identical in size and previews on asset browser, so i thought its fine.